src/Ombi.Schedule/Jobs/Ombi/NewsletterJob.cs

```diff
@@ -581,26 +635,26 @@ private void CreateMovieHtmlContent(StringBuilder sb, MovieResponseDto info)
                     {
                         continue;
                     }
+
                     int.TryParse(t.TvDbId, out var tvdbId);
                     var info = await _tvApi.ShowLookupByTheTvDbId(tvdbId);
                     if (info == null)
                     {
                         continue;
                     }
+
                     var banner = info.image?.original;
                     if (!string.IsNullOrEmpty(banner))
                     {
                         banner = banner.Replace("http", "https"); // Always use the Https banners
                     }
-                    AddImageInsideTable(sb, banner);
-
-                    sb.Append("<tr>");
-                    sb.Append(
-                        "<td align=\"center\" style=\"font-family: sans-serif; font-size: 14px; vertical-align: top;\" valign=\"top\">");
 
-                    Href(sb, $"https://www.imdb.com/title/{info.externals.imdb}/");
-                    Header(sb, 3, t.Title);
-                    EndTag(sb, "a");
+                    //GET BACKGROUND URL HERE AND CALL 
+                    AddBackgroundInsideTable(sb, $"https://image.tmdb.org/t/p/w1280/");
+                    AddPosterInsideTable(sb, banner);
```

Comment:
Need to fetch TV background before this call

Reply:
Done